leakcanary-android-core/src/main/res/values/leak_canary_strings.xml

```diff
@@ -23,8 +23,13 @@
   and open sourced by <a href="https://twitter.com/Piwai">Pierre-Yves Ricau</a> at <a href="https://square.github.io">Square</a>.<br><br>
   You can learn more about memory leaks at <a href="https://squ.re/leaks">https://squ.re/leaks</a>.<br><br>
   We welcome contributions from the community - please do not hesitate to
-  <a href="https://github.com/square/leakcanary/issues">report an issue</a> or open a pull request!<br>
+  <a href="https://github.com/square/leakcanary/issues">report an issue</a> or open a pull request!<br><br>
 ]]></string>
+  <string name="leak_canary_heap_dump_disabled_text">Heap dumping is currently disabled, because <i>%s</i>.</string>
+  <string name="leak_canary_heap_dump_enabled_text">Heap dumping is currently enabled</string>
+  <string name="leak_canary_heap_dump_disabled_by_app">dumpHeap was set to false in LeakCanary.Config by the Application</string>
```

Comment:
Nit: not `by Application`, more like `by developer` 😄 I think we can just drop this part and end sentence on `in LeakCanary.Config`

Reply:
Agreed, done :)